php/razorpay_js/index.html

```diff
@@ -0,0 +1,402 @@
+						);
+						openRPCheckout(
+								newRazorpayCustomer,
+								orderDetails.id,
+								orderDetails.amount,
+								orderDetails.currency
+						);
+					}
+				}
+				console.info(
+				"========= Payment Done =========== "
+				);
+			} catch (err) {
+				payButton.classList.remove('disable')
+				console.log("Error:", err);
+			}
+		}
+
+		async function createChargebeeCustomer(customerDetail) {
+			console.info(
+					"====== Creating CB Customer =========="
+			);
+			let myHeaders = new Headers();
+			myHeaders.append("Content-Type", "application/x-www-form-urlencoded");
+
+			let urlencoded = new URLSearchParams();
+			urlencoded.append("first_name", customerDetail.first_name || '');
+			urlencoded.append("last_name", customerDetail.last_name || '');
+			urlencoded.append("email", customerDetail.email || '');
+
+			let requestOptions = {
+				method: 'POST',
+				headers: myHeaders,
+				body: urlencoded,
+			};
+
+			let newCbCustomer = await fetch("http://localhost:3000/api/create_cb_customer", requestOptions)
+					.then(response => response.json())
+					.then(result => result)
+					.catch(error => console.log('error', error));
+			return newCbCustomer;
+		}
+
+		async function createCbEstimate() {
+			console.info(
+				"====== Creating CB Subscription for Items Estimation =========="
+			);
+			try {
+				let myHeaders = new Headers();
+				myHeaders.append("Content-Type", "application/x-www-form-urlencoded");
+
+				let urlencoded = new URLSearchParams();
+				urlencoded.append(
+					"subscription_items[item_price_id][0]",
+					ITEM_PRICE_ID
+				);
+				urlencoded.append("subscription_items[billing_cycles][0]", 2);
+				urlencoded.append("subscription_items[quantity][0]", 1);
+				urlencoded.append("cb_customer_id", CB_CUSTOMER_ID);
+
+				let requestOptions = {
+					method: "POST",
+					headers: myHeaders,
+					body: urlencoded,
+				};
+
+				let estimate = await fetch(
+					"http://localhost:3000/api/create_estimate",
+					requestOptions
+				)
+					.then((response) => response.json())
+					.then((result) => {
+						console.log(result);
+						return result;
+					})
+					.catch((error) => console.log("error", error));
+				return estimate;
+			} catch (error) {
+				console.log("error ========", error);
+			}
+		}
+
+		async function createRazorpayCustomer(customerData) {
+			console.info(
+				`======= Creating New Customer in Razorpay ========`
+			);
+			try {
+				let myHeaders = new Headers();
+				myHeaders.append("Content-Type", "application/json");
+
+				let urlencoded = {
+					name: customerData.name || "Test User 14",
+					contact: customerData.contact || "9123456782",
+					email: customerData.email || "testuser14@example.com"
+				};
+
+				let requestOptions = {
+					method: "POST",
+					headers: myHeaders,
+					body: JSON.stringify(urlencoded),
+				};
+
+				let newRazorpayCustomer = await fetch(
+					"http://localhost:3000/api/create_razorpay_customer",
```

Comment:
Extract the server URL out to a global variable, then build the URLs while making the request